tests/unit/test-util/test.adb.js

```diff
@@ -236,6 +238,52 @@ describe('utils/adb', () => {
       sinon.assert.calledOnce(adb.util.readAll);
       assert.deepEqual(packages, ['org.mozilla.fennec', 'org.mozilla.firefox']);
     });
+
+    it('resolves only the given firefox APK', async () => {
```

Comment:
It seems that these two new test cases are both testing same "exact match" scenario, it doesn't seem that we actually need both (at least if I'm not missing something that makes these two tests actually different).

If we feel that we need to test explicitly this scenario on both "org.mozilla.fennec" and "com.some.firefox.fork", we could at least extract one of these test functions into an helper function which takes the package name as a parameter and then we can call it for both these package names in the same test case.

@khaled-cliqz how that sounds to you?